### PR TITLE
DHFPROD-10180: Review skipped test for exploreXMLValidations.cy.tsx

### PR DESCRIPTION
**Pull request by amejiaq**
#### Checklist: 
```diff
- Note: do not change the below
```

- ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
- [x] Ran newly added/edited cypress tests on Firefox locally
  

- ##### Reviewer:

- [x] Reviewed Tests
- [x] Added to Release Wiki

### CODE REVIEW

**Review by aniamy — DISMISSED** [review]

**Review by aniamy — DISMISSED** [review]

**Comment by aniamy** [comment]
Improvements: We do not clean localestorage before each test. So we can remove lines that search for "Bowman" in every it. And just do it in the first it and assume is there in the next ones.

File: marklogic-data-hub-central/ui/e2e/cypress/e2e/explore/exploreXMLValidations.cy.tsx [hunk 1]
```diff
@@ -247,8 +251,11 @@ describe("xml scenario for snippet view on browse documents page", () => {
     browsePage.getSearchText().clear();
   });
 
-  it.skip("Verify metadata view of the document", () => {
+  it("Verify metadata view of the document", () => {
+    browsePage.clickTableView();
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/e2e/cypress/e2e/explore/exploreXMLValidations.cy.tsx:255]: Aren't we still in table view at this point?

File: marklogic-data-hub-central/ui/e2e/cypress/e2e/explore/exploreXMLValidations.cy.tsx [hunk 2]
```diff
@@ -247,8 +251,11 @@ describe("xml scenario for snippet view on browse documents page", () => {
...
+    cy.waitForAsyncRequest();
     browsePage.getSearchText().clear();
+    browsePage.getApplyFacetsButton().click();
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/e2e/cypress/e2e/explore/exploreXMLValidations.cy.tsx:258]: Why are we appling faces here? Read my suggestion of removing all this clear and re-searching for "Bowman". Looks like it is not needed.

**Comment by amejiaq** [comment]
🛠Thanks @aniamy  for the advice, when I was doing the ticket that it was headache, but the trick was avoid clear session store after each test.

File: marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx [hunk 3]
```diff
@@ -338,6 +338,10 @@ class BrowsePage {
     return cy.findByPlaceholderText("Search").clear();
   }
 
+  getSearchTextElement() {
+    return cy.findByPlaceholderText("Search");
+  }
```
Comment (vmendez-marklogic) [inline marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx:343]: image

image


The getSearchText getter should be replaced in all files that is being used, it is not a good idea to duplicate getters. The getSearchTextElement is bassically the same that getSearchText, so everything should be change to getSearchTextElement and then delete getSearchText() getter.

**Review by vmendez-marklogic — CHANGES_REQUESTED** [review]

File: marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx (hunk 3 above)
Comment (amejiaq) [reply marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx:343]: The issue with the original function has added a clear function at the end, if I change that function in all files maybe I can cause a lot of e2e failures maybe we can create a ticket to refactor all of that ?

**Review by Sanjeevani19 — COMMENTED** [review]
@amejiaq there are a couple of conflicts, please resolve and re-request review

**Review by aniamy — APPROVED** [review]

**Review by vmendez-marklogic — DISMISSED** [review]

**Review by aniamy — DISMISSED** [review]

**Review by aniamy — DISMISSED** [review]

**Review by vmendez-marklogic — APPROVED** [review]

**Pull request merged**